pkg/operator/baremetal_pod.go

```diff
@@ -362,8 +483,11 @@ func createContainerMetal3IronicConductor(config *OperatorConfig, baremetalProvi
 		SecurityContext: &corev1.SecurityContext{
 			Privileged: pointer.BoolPtr(true),
 		},
-		Command:      []string{"/bin/runironic-conductor"},
-		VolumeMounts: volumeMounts,
+		Command: []string{"/bin/runironic-conductor"},
+		VolumeMounts: []corev1.VolumeMount{
+			sharedVolumeMount,
+			inspectorCredentialsMount,
+		},
 		Env: []corev1.EnvVar{
 			setMariadbPassword(),
 			buildEnvVar("HTTP_PORT", "http_port", baremetalProvisioningConfig),
```

Comment:
Does the ironicConductor container need the HtpasswdEnvVar to be passed in or is it only required for other containers talking to the ironicConductor container?

Reply:
The only incoming connection to the conductor is the json-rpc one, which I haven't attempted to implement basic-auth for here (mainly because https://review.opendev.org/742474 isn't available in a released ironic, so we'd have to do some really ugly stuff).
So you're completely correct that we'll need to pass an HtpasswdEnvVar here eventually, but when we do we'll create a new secret for it.